Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -377,7 +377,7 @@ void NetworkStorageManager::spaceGrantedForOrigin(const WebCore::ClientOrigin& o
     if (m_totalUsage)
         m_totalUsage = *m_totalUsage + amount;
 
-    if (!m_totalUsage || *m_totalUsage > m_totalQuota)
+    if (!m_totalUsage || !m_totalQuota || *m_totalUsage > *m_totalQuota)
```

Comment:
m_totalQuota should already be initialized at this point, so the null check is unnecessary.